.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+    mocha: true
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module'
+  },
+  plugins: ['@typescript-eslint', 'prettierx'],
+  extends: [
+    'standard',
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettierx/standardx',
+    'plugin:prettierx/@typescript-eslint'
+  ],
+  rules: {
+    'no-void': 'off',
+
+    // Disabled because we intentionally have any for generics
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
I really prefer to have that one disabled at a line level is TS files. 
One of the purpose of TS is to have strong typing, so using 'any' shall remain a documented exception in the code.

Reply:
Yeah, okay I will bring this one to line level